lib/board.rb

```diff
@@ -97,4 +97,26 @@ def place(ship, coordinates)
       "Sorry, not valid placement"
     end
   end
+
+  def render(cheat = false)
+    evaluate(cheat)
+    print_board
+  end
+
+  def evaluate(cheat)
+    @board = ""
+    @cells.each_value do |value|
+      @board << value.render(cheat)
```

Comment:
This thought occurred to me last night after wrapping up! When I see a shovel operator, while I know << is a method for strings, my mind automatically resorts to the object on the left being an array. And so I think if we used a += operator to concatenate the string, @board might read more like a string object instead of an array.

Reply:
Good call. I didn't know you could use += on strings. it definitely makes it more clear.